prime-dotnet-webapi/Services/DefaultEnrolleeService.cs

```diff
@@ -391,10 +395,6 @@ private async Task<EnrolmentStatus> CreateEnrolmentStatusInternalAsync(int enrol
             }
 
             var created = await _context.SaveChangesAsync();
```

Comment:
Since we are removing the "created" check, this line can just be 
`await _context.SaveChangesAsync();`